ci/test-checks.sh

```diff
@@ -7,6 +7,18 @@ set -e
 
 cd "$(dirname "$0")/.."
 
+usage() {
+  cat <<EOF
+usage: $0 [-h] [-l]
+
+Run sanity check for solana build
+
+  -h              Show help
+  -l              Run sanity check for local repo (allow uncommitted changes)
+
+EOF
+}
+
```

Comment:
If we check $CI below, I think we might not need this anymore. 🤔

Reply:
yeah. updated.